tools/fetch-emoji-names-list

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+set -x
+
+FILENAME="zulipterminal/emoji_names.py"
+SCRIPT="tools/convert-emoji-names-list"
+
+curl https://raw.githubusercontent.com/zulip/zulip/master/tools/setup/emoji/emoji_names.py > $FILENAME
+
+sed -i '1d' $FILENAME  # delete first line containing imports
+sed -i 's/EMOJI_NAME_MAPS = //g' $FILENAME  # delete variable name
+python3 $SCRIPT $FILENAME
```

Comment:
I see you're using the script here, but but was a little confused as I thought you were using python instead of `sed`?